{
  "build": "py312haa95532_0",
  "build_number": 0,
  "channel": "https://repo.anaconda.com/pkgs/main/win-64",
  "constrains": [],
  "depends": [
    "python >=3.12,<3.13.0a0"
  ],
  "extracted_package_dir": "C:\\Users\\ADITI\\.conda\\pkgs\\exceptiongroup-1.2.0-py312haa95532_0",
  "files": [
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
    "Lib/site-packages/exceptiongroup/__init__.py",
    "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-312.pyc",
    "Lib/site-packages/exceptiongroup/_catch.py",
    "Lib/site-packages/exceptiongroup/_exceptions.py",
    "Lib/site-packages/exceptiongroup/_formatting.py",
    "Lib/site-packages/exceptiongroup/_suppress.py",
    "Lib/site-packages/exceptiongroup/_version.py",
    "Lib/site-packages/exceptiongroup/py.typed"
  ],
  "fn": "exceptiongroup-1.2.0-py312haa95532_0.conda",
  "license": "MIT",
  "link": {
    "source": "C:\\Users\\ADITI\\.conda\\pkgs\\exceptiongroup-1.2.0-py312haa95532_0",
    "type": 1
  },
  "md5": "6953b8470649ce4e63e14329905ecf15",
  "name": "exceptiongroup",
  "package_tarball_full_path": "C:\\Users\\ADITI\\.conda\\pkgs\\exceptiongroup-1.2.0-py312haa95532_0.conda",
  "paths_data": {
    "paths": [
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
        "path_type": "hardlink",
        "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "sha256_in_prefix": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "size_in_bytes": 5
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
        "path_type": "hardlink",
        "sha256": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "sha256_in_prefix": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "size_in_bytes": 3704
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
        "path_type": "hardlink",
        "sha256": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "sha256_in_prefix": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "size_in_bytes": 6603
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
        "path_type": "hardlink",
        "sha256": "3d2139457c7f11dac69e0fa043645f23fbe4d61e1664a8520384d0f73064ff6f",
        "sha256_in_prefix": "3d2139457c7f11dac69e0fa043645f23fbe4d61e1664a8520384d0f73064ff6f",
        "size_in_bytes": 1541
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
        "path_type": "hardlink",
        "sha256": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "sha256_in_prefix": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "size_in_bytes": 81
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
        "path_type": "hardlink",
        "sha256": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "sha256_in_prefix": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "size_in_bytes": 94
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__init__.py",
        "path_type": "hardlink",
        "sha256": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "sha256_in_prefix": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "size_in_bytes": 1049
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "69ea5c1528707d83a49d4ef92a4facca6801f772ea4891a8f4a786738fe7e90b",
        "sha256_in_prefix": "69ea5c1528707d83a49d4ef92a4facca6801f772ea4891a8f4a786738fe7e90b",
        "size_in_bytes": 1018
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "a70b42a602d4abe14c06dd432c759f287be29ea5e1f27743a3d9a19ef547ca87",
        "sha256_in_prefix": "a70b42a602d4abe14c06dd432c759f287be29ea5e1f27743a3d9a19ef547ca87",
        "size_in_bytes": 4764
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "b1de3013b64792e26c05f81925066d7001326f20dcfe654e3d84886f7ca365a3",
        "sha256_in_prefix": "b1de3013b64792e26c05f81925066d7001326f20dcfe654e3d84886f7ca365a3",
        "size_in_bytes": 13675
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "c5881dac6e2e1886132fa19d673f517d2a6dc751a4414b692c389c302636e237",
        "sha256_in_prefix": "c5881dac6e2e1886132fa19d673f517d2a6dc751a4414b692c389c302636e237",
        "size_in_bytes": 21911
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "773cc92d0d59dd754cb0c58dd1f367f81bb5891a393e71b0bb01a3b2058a6f37",
        "sha256_in_prefix": "773cc92d0d59dd754cb0c58dd1f367f81bb5891a393e71b0bb01a3b2058a6f37",
        "size_in_bytes": 1302
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-312.pyc",
        "path_type": "hardlink",
        "sha256": "b18358ca9fa7163548b6476063a7af5aa2bed736d43445b28e540ae99350138d",
        "sha256_in_prefix": "b18358ca9fa7163548b6476063a7af5aa2bed736d43445b28e540ae99350138d",
        "size_in_bytes": 257
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_catch.py",
        "path_type": "hardlink",
        "sha256": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "sha256_in_prefix": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "size_in_bytes": 4669
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_exceptions.py",
        "path_type": "hardlink",
        "sha256": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "sha256_in_prefix": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "size_in_bytes": 11010
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_formatting.py",
        "path_type": "hardlink",
        "sha256": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "sha256_in_prefix": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "size_in_bytes": 21005
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_suppress.py",
        "path_type": "hardlink",
        "sha256": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "sha256_in_prefix": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "size_in_bytes": 1338
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_version.py",
        "path_type": "hardlink",
        "sha256": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "sha256_in_prefix": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "size_in_bytes": 164
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/py.typed",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      }
    ],
    "paths_version": 1
  },
  "requested_spec": "None",
  "sha256": "d03432b12d660e8b8b9a3c28cbe7d955748bbb62406dc3a43dbabd3614a2dda7",
  "size": 38202,
  "subdir": "win-64",
  "timestamp": 1706031664000,
  "url": "https://repo.anaconda.com/pkgs/main/win-64/exceptiongroup-1.2.0-py312haa95532_0.conda",
  "version": "1.2.0"
}